hdl/modules/fofb_sys_id/cheby/wb_fofb_sys_id_regs.cheby

```diff
@@ -7,24 +7,160 @@ memory-map:
               access: ro
               address: next
               description: |
                 Maximum number of BPM positions that can be flatenized per axis
+                (x or y)
+    - block:
+        name: prbs
+        description: Interface to PRBS-related regs
+        address: next
+        children:
           - reg:
-              name: base_bpm_id
-              width: 8
+              name: ctl
+              width: 32
               access: rw
               address: next
               description: |
-                Together with max_num_cte, defines the range of BPM position
-                indexes being flatenized, which is given by
-                [base_bpm_id, base_bpm_id + max_num_cte) -> BPM x positions; and
-                [base_bpm_id + 256, base_bpm_id + 256 + max_num_cte) -> BPM y
-                positions. The valid range of this register is [0, 255].
+                PRBS distortion control register
+              children:
+                - field:
+                    name: rst
+                    x-hdl:
+                      type: autoclear
+                    range: 0
+                    description: |
+                      Resets PRBS
+
+                      write 0: no effect
+                      write 1: resets PRBS (this bit autoclears)
+                - field:
+                    name: step_duration
+                    range: 10-1
+                    description: |
+                      Duration of each PRBS step in FOFB cycles (max: 0x3FF)
+
+                      write 0x000: new PRBS iteration at each FOFB cycle
+                      write 0x001: new PRBS iteration at each 2 FOFB cycles
+                      ...
+                      write 0x3FF: new PRBS iteration at each 1024 FOFB cycles
+                - field:
+                    name: lfsr_length
+                    range: 15-11
+                    description: |
+                      Length of internal LFSR (max: 0x1E)
+                      Together with step_duration, defines the duration of PRBS,
+                      which is given by [2^(lfsr_length + 2) - 1]*(step_duration + 1)
+
+                      write 0x00: set LFSR length to 2
+                      write 0x01: set LFSR length to 3
+                      ...
+                      write 0x1E: set LFSR length to 32
+                - field:
+                    name: bpm_pos_distort_en
+                    range: 16
+                    description: |
+                      Enables/disables PRBS-based distortion on BPM positions
+
+                      write 0: distortion disabled
+                      write 1: distortion enabled
+                - field:
+                    name: sp_distort_en
+                    range: 17
+                    description: |
+                      Enables/disables PRBS-based distortion on setpoints
+
+                      write 0: distortion disabled
+                      write 1: distortion enabled
```

Comment:
Can they be enabled simultaneously? If not, that should be documented here.

Reply:
They can, though I don't know how useful it is.

Reply:
They can, @ericonr .